### ISSUE TITLE
raw_data endpoint missing

### ISSUE THREAD

**Issue by rfs-** [AHAAAAAAA/PokemonGo-Map#2335]
I've installed prereq's by following the video walkthrough and can see in the cmd that the server is running and populating but when I check on a web link (actually on a web server not localhost) the map loads but in the console it says (*website domain replaced)*: 

GET http://website.com/poke/raw_data?pokemon=true&pokestops=true&gyms=true&scanned=false 404 (Not Found)

I see that a pogom.db file is constantly being updated so I assume that there raw_data endpoint is supposed to pull from that pogom.db but I do not see a raw_data file or variable?

**Comment by wcastello** [comment]
Does /poke works?

**Comment by rfs-** [comment]
Yes, i basically took the map.html from templates and put it in the root of everything as index.html since it was referencing assets as if it was in root. The map shows up correctly with the right location using my google api key but no pokestops, gyms or pokemon show up and I get a 404 error referencing the raw_data endpoint.

**Comment by rfs-** [comment]
is raw_data a placeholder for a file that I have to write to connect to pogom.db? Seems weird that it's an ajax request to an endpoint with no extension

**Comment by spybe** [comment]
Hi @rfs- 
Have you find a solution since? I got the exact same issue...
